src/main/java/com/liveramp/daemon_lib/executors/JobletExecutors.java

```diff
@@ -40,13 +41,17 @@
     private static final int DEFAULT_POLL_DELAY = 1000;
 
     public static <T extends JobletConfig> ForkedJobletExecutor<T, JobletConfigMetadata, Integer> get(DaemonNotifier notifier, String tmpPath, int maxProcesses, Class<? extends JobletFactory<T>> jobletFactoryClass, Map<String, String> envVariables, JobletCallback<? super T> successCallback, JobletCallback<? super T> failureCallback, ProcessJobletRunner<Integer> jobletRunner) throws IOException, IllegalAccessException, InstantiationException {
+      return get(notifier, tmpPath, maxProcesses, jobletFactoryClass, envVariables, successCallback, failureCallback, jobletRunner, JobletConfigStorage.DEFAULT_SERIALIZER, JobletConfigStorage.getDefaultDeserializer());
+    }
+
+    public static <T extends JobletConfig> ForkedJobletExecutor<T, JobletConfigMetadata, Integer> get(DaemonNotifier notifier, String tmpPath, int maxProcesses, Class<? extends JobletFactory<T>> jobletFactoryClass, Map<String, String> envVariables, JobletCallback<? super T> successCallback, JobletCallback<? super T> failureCallback, ProcessJobletRunner<Integer> jobletRunner, Function<? super T, byte[]> serializer, Function<byte[], T> deserializer) throws IOException, IllegalAccessException, InstantiationException {
```

Comment:
Only 245 params more to go before we hit the JVM limit.

Reply:
![image](https://user-images.githubusercontent.com/706280/29722462-c5691ef2-8975-11e7-9883-007ad03eead8.png)

Reply:
If we run out of params, we can write the "full" sig in a comment and write a ruby script to parse the comment into a series of `Pair`s with the appropriate generic types ◉ ͜ʖ ◉